phasmophobia_evidence_v3/widget.js

```diff
@@ -352,31 +354,63 @@ window.addEventListener("onWidgetLoad", function (obj) {
         modOrVIPPermission(config),
         data,
         _setCounterName,
-        [data.text]
+        [1, data.text]
```

Comment:
This is a "magic number". It should be made into a constant above, so that if there is ever a change, it only has to be changed in one location.

Reply:
Should be all set (assuming I understood your request)